sw/airborne/modules/ctrl/follow_me.c

```diff
@@ -130,8 +130,8 @@ void follow_me_periodic(void)
     // Rotate the waypoint
     float cos_heading = cosf(diff_targetpos_heading/180.*M_PI);
     float sin_heading = sinf(diff_targetpos_heading/180.*M_PI);
-    wp_new_enu.x = ((wp_new_enu.x - cur_targetpos.x) * cos_heading) + ((wp_new_enu.y - cur_targetpos.y) * sin_heading) +  + cur_targetpos.x;
-    wp_new_enu.y = (-(wp_new_enu.y - cur_targetpos.y) * sin_heading) + ((wp_new_enu.y - cur_targetpos.y) * cos_heading) + cur_targetpos.y;
+    wp_new_enu.x = ((wp_new_enu.x - cur_targetpos.x) * cos_heading) + ((wp_new_enu.y - cur_targetpos.y) * sin_heading) + cur_targetpos.x;
+    wp_new_enu.y = (-(wp_new_enu.x - cur_targetpos.x) * sin_heading) + ((wp_new_enu.y - cur_targetpos.y) * cos_heading) + cur_targetpos.y;
```

Comment:
This is not 100% correct I think because the `cur_targetpos` is actually not the GPS position but a defined position around our GPS (for example the back of the deck of a ship). This means that if that point is far away from the actually GPS position the rotation is also from the incorrect spot.